### PR TITLE
[CLOUDTRUST-3208] URL management for email action

### PR DESCRIPTION
**Pull request by fperot74**

### CODE REVIEW

**Review by harture — CHANGES_REQUESTED** [review]

File: configs/keycloak_bridge.yml [hunk 1]
```diff
@@ -63,6 +63,10 @@ trustid-groups:
 # Keycloak configs
 keycloak-api-uri: http://localhost:8080
 keycloak-oidc-uri: http://localhost:8080 http://127.0.0.1:8080
```
[the remark this answers is not in the document]
Comment (fperot74) [reply configs/keycloak_bridge.yml]: Done

File: cmd/keycloakb/keycloak_bridge.go [hunk 2]
```diff
@@ -373,11 +396,7 @@ func main() {
 	commonKcAdaptor := keycloakb.NewKeycloakAuthClient(keycloakClient, logger)
 
 	// Public Keycloak URL
-	var keycloakPublicURL string
-	{
-		urls := strings.Split(keycloakConfig.AddrTokenProvider, " ")
-		keycloakPublicURL = urls[0]
-	}
+	var keycloakPublicURL = keycloakConfig.AddrTokenProvider[0]
```
[the remark this answers is not in the document]
Comment (fperot74) [reply cmd/keycloakb/keycloak_bridge.go:391]: There is no change here. We already was using only the first item

[the remark this answers is not in the document]
Comment (harture) [reply cmd/keycloakb/keycloak_bridge.go:391]: I think we should not use it anymore. As it always depends of the realm

File: cmd/keycloakb/keycloak_bridge.go [hunk 3]
```diff
@@ -1191,7 +1200,7 @@ func main() {
 		var route = mux.NewRouter()
 
 		// Version.
-		route.Handle("/", http.HandlerFunc(commonhttp.MakeVersionHandler(keycloakb.ComponentName, ComponentID, keycloakb.Version, Environment, GitCommit)))
+		route.Handle("/", commonhttp.MakeVersionHandler(keycloakb.ComponentName, ComponentID, keycloakb.Version, Environment, GitCommit))
```
Comment (harture) [inline cmd/keycloakb/keycloak_bridge.go:1203]: Why we can now remove it ? is it an "error" from the beginning or a new feature from http ?

**Review by harture — APPROVED** [review]

**Pull request merged**